Tests/AK/TestComplex.cpp

```diff
@@ -9,9 +9,47 @@
 #include <AK/Complex.h>
 
 static_assert([]() {
+    auto const z = Complex<double>();
     auto const r = complex_real_unit<double>;
     auto const i = complex_imag_unit<double>;
 
+    auto basic_math = [&](auto const& c) {
+        auto n = c;
+        assert(c == c);
```

Comment:
we don't use `assert()` here, either use `VERIFY()` or the LibTest macros `EXPECT_EQ()` and friends (I'd suggest the LibTest macros)

Reply:
I didn't knew what to use here, so I used assert, so it mirrors `static_assert`. If `STATICTEST_CASE` was introduced it makes sence to use these macros indeed.

Reply:
in general, we don't use the C `assert` in serenity code at all, I'm pretty sure it's not rewritten by LibTest so it'll actually crash instead of showing a nice error here.
I'd suggest using EXPECT_EQ here.

Reply:
We are safe to use it there:
https://github.com/SerenityOS/serenity/blob/444b2d9ec2e962235e2ea7c15c8457c8627d7ca2/Userland/Libraries/LibTest/Macros.h#L25-L32

Reply:
Yes, as I said, VERIFY and LibTest macros are okay, just that `assert()` isn't.